src/ProjectTemplates/Web.ProjectTemplates/RazorPagesWeb-CSharp.csproj.in

```diff
@@ -15,11 +13,10 @@
     <None Update="app.db" CopyToOutputDirectory="PreserveNewest" ExcludeFromSingleFile="true" />
   </ItemGroup>
   <!--#endif -->
-  <!--#if (IndividualB2CAuth || IndividualLocalAuth || OrganizationalAuth || RazorRuntimeCompilation) -->
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.Authentication.AzureAD.UI" Version="${MicrosoftAspNetCoreAuthenticationAzureADUIPackageVersion}" Condition="'$(OrganizationalAuth)' == 'True'" />
-    <PackageReference Include="Microsoft.AspNetCore.Authentication.AzureADB2C.UI" Version="${MicrosoftAspNetCoreAuthenticationAzureADB2CUIPackageVersion}" Condition="'$(IndividualB2CAuth)' == 'True'" />
+    <PackageReference Include="Microsoft.Identity.Web" Version="${MicrosoftIdentityWebPackageVersion}" />
```

Comment:
TODO: This probably shouldn't be included unconditionally, Looks like this would only be used for IndividualB2CAuth and OrganizationalAuth? @jmprieur

Reply:
There is a bug for IndividualAuth when using AAD as an external provider, but otherwise this is supported.

Reply:
What I mean here is that we should only include these packages in OrganizationalAuth and IndividualB2CAuth scenarios right? In all other cases, these packages won't be used at all and should not be included in the csproj, even if they are harmless.

Reply:
when you write these pacakges? do you mean AzureAD.UI and AzureADB2C.UI? 
the goal was to not depend on them.